dbms/src/Flash/Planner/Plans/PhysicalAggregationBuild.cpp

```diff
@@ -48,18 +43,21 @@ void PhysicalAggregationBuild::buildPipelineExecGroup(
         context.getSettingsRef().max_spilled_rows_per_file,
         context.getSettingsRef().max_spilled_bytes_per_file,
         context.getFileProvider());
-
     auto params = AggregationInterpreterHelper::buildParams(
         context,
         before_agg_header,
         concurrency,
-        concurrency,
+        1,
```

Comment:
fix, for non fine grained shuffle, the actual value is 1.
https://github.com/pingcap/tiflash/blob/99e4b2050396fdc41ef5fde4e8e0c9bc12b37472/dbms/src/Flash/Planner/Plans/PhysicalAggregation.cpp#L110